public/app/core/components/NavBar/NavBarItemMenuItem.tsx

```diff
@@ -45,8 +45,17 @@ export function NavBarItemMenuItem({ item, state, onNavigate }: NavBarItemMenuIt
     ref
   );
 
+  const { keyboardProps } = useKeyboard({
+    onKeyDown: (e) => {
+      if (e.key === 'ArrowLeft') {
+        onClose();
```

Comment:
I don't think it really matters in this case, but should we add `preventDefault` here for "correctness"?

Reply:
btw @kaydelaney I think in this case is not needed to add the `prevenDefault`, we are using `useKeyboard` and the hook is doing it automatically.